timelock-impl/src/main/java/com/palantir/atlasdb/timelock/batch/MultiClientConjureTimelockResource.java

```diff
@@ -74,6 +77,28 @@ public static MultiClientConjureTimelockService jersey(
                 MoreExecutors.directExecutor()));
     }
 
+    @Override
+    public ListenableFuture<Map<Namespace, ConjureStartTransactionsResponse>> startTransactions(
+            AuthHeader authHeader, Map<Namespace, ConjureStartTransactionsRequest> requests) {
+        List<ListenableFuture<Map.Entry<Namespace, ConjureStartTransactionsResponse>>> futures = KeyedStream.stream(
+                        requests)
+                .map(this::startTransactionsForSingleNamespace)
+                .values()
+                .collect(Collectors.toList());
+        return handleExceptions(() ->
+                Futures.transform(Futures.allAsList(futures), ImmutableMap::copyOf, MoreExecutors.directExecutor()));
```

Comment:
[not actionable now]: I'm fine with this, though maybe we should explore at some point whether the futures are costing us anything. I don't think it will, but just in case...